s3repo/view.py

```diff
@@ -0,0 +1,26 @@
+from flask.views import MethodView
+from flask import render_template, Response
+import os
+
+class IndexView(MethodView):
+    """IndexView is just methods to display listings or download
+    files with browser.
+    """
+    def get_directory(self, subpath, files, folders):
```

Comment:
I think the class should have one public method get and two private ("get_directory" and "get_file").

Reply:
Done. See the patch about s3view.